### ISSUE TITLE
[v2.8] Rancher logging does not work on windows nodes when using `logging-operator:4.4.0`

### ISSUE THREAD

**Issue by HarrisonWAffel** [rancher/rancher#45308]
**Rancher Server Setup**
- Rancher version: v2.8-head, 2.9-head
- Installation option (Docker install/Helm Chart): n/a
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc): n/a
- Proxy/Cert Details: n/a

**Information about the Cluster**
- Kubernetes version: n/a
- Cluster Type (Local/Downstream):
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider):

**User Information**
- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom)
  - If custom, define the set of permissions:

**Describe the bug**

It appears that the bump to 4.4.0 in the rancher logging chart in dev-2.8 and dev-v2.9 has broken windows support as the windows fluent bit agent version has not been bumped to the 2.x line. This results in the following errors related to the arguments passed to the agent via the fluent bit configuration

```
CrashLoopBackOff (back-off 40s restarting failed container=fluent-bit pod=rancher-logging-root-win-agent-fluentbit-tplgf_cattle-logging-system(a020e789-9667-4622-83ef-fa101ba931f8)) | Last state: Terminated with -1: Error, started: Tue, Apr 30 2024 12:08:22 pm, finished: Tue, Apr 30 2024 12:08:24 pm
```

```
[engine] started (pid=5860)
2024-04-30T19:09:32.849260100Z [2024/04/30 19:09:32] [ info] [storage] version=1.1.5, initializing...
2024-04-30T19:09:32.849378600Z [2024/04/30 19:09:32] [ info] [storage] in-memory
2024-04-30T19:09:32.849378600Z [2024/04/30 19:09:32] [ info] [storage] normal synchronization mode, checksum disabled, max_chunks_up=128
2024-04-30T19:09:32.850580500Z [2024/04/30 19:09:32] [ info] [cmetrics] version=0.2.2
2024-04-30T19:09:32.875165500Z [2024/04/30 19:09:32] [error] [config] kubernetes: unknown configuration property 'Kube_Token_TTL'. The following properties are allowed: buffer_size, tls.debug, tls.verify, tls.vhost, merge_log, merge_parser, merge_log_key, merge_log_trim, keep_log, kube_url, kube_meta_preload_cache_dir, kube_ca_file, kube_ca_path, kube_tag_prefix, kube_token_file, kube_token_command, labels, annotations, k8s-logging.parser, k8s-logging.exclude, use_journal, regex_parser, dummy_meta, dns_retries, dns_wait_time, cache_use_docker_id, use_tag_for_meta, use_kubelet, kubelet_port, and kube_meta_cache_ttl.
2024-04-30T19:09:32.875165500Z [2024/04/30 19:09:32] [ help] try the command: fluent-bit -F kubernetes -h
2024-04-30T19:09:32.875165500Z 
2024-04-30T19:09:32.875165500Z [2024/04/30 19:09:32] [ info] [input] pausing tail.0
[2024/04/30 19:09:32] [error] [lib] backend failed
```

**To Reproduce**
deploy a rancher logging version using `logging-operator:4.4.0` to a windows cluster (`103.1.0+up4.4.0`, `104.0.0-rc1+up4.4.0`)

**Result**
The windows agent pod enters a crash loop

**Expected Result**
No Crashloop

**Screenshots**

**Additional context**
This does not reproduce for the 2.7 logging chart 
relates to https://github.com/rancher/windows/issues/222

**Comment by HarrisonWAffel** [comment]
Note that this fix is included in `103.1.1-rc1`, which means that rancher must be configured to deploy pre-release charts in order for this to be tested

**Comment by slickwarren** [comment]
tested on v2.8-head (4a2b4fe):
* deploy logging 103.1.1-rc1 on a cluster with windows nodes (1.27.13+rke2r1) -- pass
  * upgrade the cluster to a later version of k8s (1.28.9+rke2r1) -- pass
